### PR TITLE
test(action): Type nthCalledWith asserts

### PR DESCRIPTION
**Pull request by mwarres**

### CODE REVIEW

File: src/docker.test.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import type { InputOptions } from "@actions/core";
```
Comment (Kurt-von-Laven) [inline src/docker.test.ts:1]: Multiple instances: let's put this right above the other `import type`.

File: src/docker.test.ts [hunk 2]
```diff
@@ -68,7 +69,9 @@ describe("Docker images", (): void => {
     }
     await docker.saveDockerImages();
 
-    expect(core.getInput).nthCalledWith(1, "key", { required: true });
+    expect(core.getInput).nthCalledWith<[string, InputOptions]>(1, "key", {
```
Comment (Kurt-von-Laven) [inline src/docker.test.ts:72]: Multiple instances: can we fit all of the arguments on the line below, or does Prettier force this awkward formatting?

**Review by Kurt-von-Laven — CHANGES_REQUESTED** [review]

File: src/docker.test.ts (hunk 2 above)
Comment (mwarres) [reply src/docker.test.ts:72]: I attempted your formatting suggestion but, alas, Prettier objected.

**Review by Kurt-von-Laven — APPROVED** [review]
**LGTM**

**Pull request merged**